models/game.rb

```diff
@@ -2,7 +2,8 @@
 require './models/share_price'
 
 class Game < Base
-  many_to_one :user
+  #many_to_one :player
+  one_to_many :user
```

Comment:
this should actually be many_to_one because only one user owns the game. the other players are stored in a different column, so change this back.